### PR TITLE
feat:  add links to legend items in `allocation-summary`

### PR DESCRIPTION
**Pull request by ChaiWithJai**
### Actual

image

### CODE REVIEW

**Review by LevelbossMike — APPROVED** [review]
this looks good to me 👏

File: ui/app/templates/components/job-page/parts/summary-legend-item.hbs [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+  ></span>
   <span class="text">
-    <span class="value" data-test-legend-value="{{@datum.className}}">{{@datum.value}}</span>
-    <span class="label">{{@datum.label}}</span>
+    <span class="value" data-test-legend-value="{{@datum.className}}">
+      {{@datum.value}}
+    </span>
+    <span>
+      {{@datum.label}}
+    </span>
```
Comment (lgfa29) [inline ui/app/templates/components/job-page/parts/summary-legend-item.hbs:11]: Auto-formater strikes again 😄 

I think this file didn't actually change?

**Review by lgfa29 — APPROVED** [review]
Just missing a changelog entry.

Also, it would good if there were a quick test around this new clicking behaviour.

File: ui/app/styles/charts/distribution-bar.scss [hunk 2]
```diff
@@ -98,8 +97,6 @@
         &.is-clickable {
           a {
             display: block;
-            text-decoration: none;
-            color: inherit;
           }
 
           &:not(.is-empty) {
```
Comment (lgfa29) [inline ui/app/styles/charts/distribution-bar.scss:102]: Maybe we can remove this rule now? The background colour change on hover may not be necessary anymore.

image

**Pull request merged**